deployment/updates/scripts/2019_01_15_update_unique_mediaspace_notifications_templates.php

```diff
@@ -0,0 +1,20 @@
+<?php
+/**
+ * @package deployment
+ */
+
+define('DEPLOYMENT_DIR', realpath(__DIR__ . '/../..'));
+
+require_once (DEPLOYMENT_DIR . '/bootstrap.php');
+
+$scriptPath = realpath(DEPLOYMENT_DIR . '/../tests/standAloneClient/exec.php');
+
+$templateUpdateXmlPath = realpath(DEPLOYMENT_DIR . '/updates/scripts/xml/notifications/2019_01_15_update_unique_mediaspace_notifications_templates.template.xml');
```

Comment:
You need to check the existence of the actual file here not the template one please fix.
2019_01_15_update_unique_mediaspace_notifications_templates.template.xml ==> 2019_01_15_update_unique_mediaspace_notifications_templates.xml

Also, please add release notes for this.

Reply:
@yossipapi , done: pointed to the right file, updated from the master, added release notes